src/server-extension/query/nft.ts

```diff
@@ -1,4 +1,30 @@
 export const passionQuery = `SELECT DISTINCT ne.issuer as id
 FROM nft_entity ne
 WHERE ne.current_owner = $1
-AND ne.current_owner != ne.issuer`
\ No newline at end of file
+AND ne.current_owner != ne.issuer`
+
+export const flippingQuery = `select
```

Comment:
Please use the correct SQL formating
- KEYWORDS UPPERCASE (SELECT, FROM, GROUP BY)
- and formatiing (use some online formatter)